### PR TITLE
Warnings in meraki_mx_content_filtering

### PR DESCRIPTION
**Pull request by y0rune**

### CODE REVIEW

File: tests/integration/targets/meraki_mx_content_filtering/tasks/main.yml [hunk 1]
```diff
@@ -3,306 +3,307 @@
...
-        - "Gambling"
-        - "Hacking"
-        - "Hate and Racism"
-        - "Illegal"
-        - "Malware Sites"
-        - "Open HTTP Proxies"
-        - "Peer to Peer"
-        - "Phishing and Other Frauds"
-        - "Proxy Avoidance and Anonymizers"
+        - "Adult"
```
Comment (y0rune) [inline tests/integration/targets/meraki_mx_content_filtering/tasks/main.yml:50]: The blocked categories has been changed. 
When I curled the categories:
```bash
curl -L --request GET \
    --url https://api.meraki.com/api/v1/networks/"$MERAKI_NET_ID"/appliance/contentFiltering/categories \
    --header 'Content-Type: application/json' \
    --header 'Accept: application/json' \
    --header 'X-Cisco-Meraki-API-Key: '"$MERAKI_API_KEY"'' |
    jq .categories[].name |
    sort -u
```
I got so much different values.

File: tests/integration/targets/meraki_mx_content_filtering/tasks/main.yml [hunk 2]
```diff
@@ -3,306 +3,307 @@
...
+        - "Advertisements"
+        - "Alcohol"
     delegate_to: localhost
-    register: category_idempotent
 
-  - assert:
-      that:
-        - category_idempotent is not changed
-        - category_idempotent.data is defined
+  # - name: Set blocked URL categories with idempotency
```
Comment (y0rune) [inline tests/integration/targets/meraki_mx_content_filtering/tasks/main.yml:55]: Problem addressed in #407

File: tests/integration/targets/meraki_mx_content_filtering/tasks/main.yml [hunk 3]
```diff
@@ -3,306 +3,307 @@
...
-
-  - debug:
-      var: blocked_category
-
-  - assert:
-      that:
-        - blocked_category.changed == True
-        - blocked_category.data.blocked_url_categories | length == 1
-        - blocked_category.data.url_category_list_size == "topSites"
+        # - blocked_category.data.url_category_list_size == "fullList"
```
Comment (y0rune) [inline tests/integration/targets/meraki_mx_content_filtering/tasks/main.yml:208]: The value `url_category_list_size` has been missing from the output. I wrote the email to the support with that case.

Comment (y0rune) [reply tests/integration/targets/meraki_mx_content_filtering/tasks/main.yml:208]: I have added the issue in the dashboard-api-python: https://github.com/meraki/dashboard-api-python/issues/198

**Comment by kbreit** [comment]
```
TASK [meraki_mx_content_filtering : Reset policies] ****************************
fatal: [localhost]: FAILED! => {"changed": false, "msg": "None is not a valid content filtering category", "response": "OK (unknown bytes)", "status": 200}
```

This used to work so I'll need to look into fixing this before merging.

**Comment by y0rune** [comment]
Yup yup! When you merge that two PRs it works (the test and the code itself)

**Comment by kbreit** [comment]
Thank you for your contribution. Merging.

**Pull request merged**